server/sonar-server/src/main/java/org/sonar/server/user/UserUpdater.java

```diff
@@ -211,10 +214,10 @@ private static void checkNotEmptyParam(@Nullable String value, String param, Lis
   private static void validateLoginFormat(@Nullable String login, List<Message> messages) {
     checkNotEmptyParam(login, LOGIN_PARAM, messages);
     if (!Strings.isNullOrEmpty(login)) {
```

Comment:
Well, that was not in the code I modified :)

Moreover, there is first the check for null or empty (which eventually adds a message to messages), then if login is not null or empty, other checks. So I would not change anything here.